api/test/h3-data/mocks/h3-fixtures.ts

```diff
@@ -0,0 +1,401 @@
+export const h3Fixtures = {
```

Comment:
I suspect that, for the purpose of testing, we should probably have some values different than 0, otherwise the result of queries will always be 0, no?

Reply:
Actually 0 is a valid value for H3 Data. What we care the most are the keys of the resultant object, which are the actual H3 Indexes. Would't make a difference setting some random values